src/admin-views/editor/panel/ticket.php

```diff
@@ -145,14 +221,198 @@ class="tribe_soft_note ticket_form_right"
 					><?php
 						echo esc_html(
 							sprintf(
-								_x( '%1$s type name shows on the front end and emailed %2$s', 'admin edit ticket panel note', 'event-tickets' ),
+								// Translators: %1$s: dynamic 'RSVP' text, %2$s: dynamic 'rsvps' text.
+								_x(
+									'%1$s type name shows on the front end and emailed %2$s',
+									'admin edit ticket panel note',
+									'event-tickets'
+								),
 								tribe_get_rsvp_label_singular( 'admin_edit_rsvp_panel_note' ),
 								tribe_get_rsvp_label_plural_lowercase( 'admin_edit_ticket_panel_note' )
 							)
-						); ?>
+						);
+						?>
 					</span>
 				</div>
-				<fieldset id="tribe_ticket_provider_wrapper" class="input_block" aria-hidden="true" >
+				<div class="input_block">
+					<label class="ticket_form_label ticket_form_left" for="ticket_start_date"><?php esc_html_e( 'Start sale:', 'event-tickets' ); ?></label>
+					<div class="ticket_form_right">
+						<input
+							autocomplete="off"
+							type="text"
+							class="tribe-datepicker tribe-field-start_date ticket_field"
+							name="ticket_start_date"
+							id="ticket_start_date"
+							value="<?php echo esc_attr( $ticket ? $start_date : null ); ?>"
+							data-validation-type="datepicker"
+							data-validation-is-less-or-equal-to="#ticket_end_date"
+							data-validation-error="<?php echo esc_attr( wp_json_encode( $start_date_errors ) ); ?>"
+						/>
+						<span class="helper-text hide-if-js"><?php esc_html_e( 'YYYY-MM-DD', 'event-tickets' ); ?></span>
+						<span class="datetime_seperator"> <?php esc_html_e( 'at', 'event-tickets' ); ?> </span>
+						<input
+							autocomplete="off"
+							type="text"
+							class="tribe-timepicker tribe-field-start_time ticket_field"
+							name="ticket_start_time"
+							id="ticket_start_time"
+							<?php echo Tribe__View_Helpers::is_24hr_format() ? 'data-format="H:i"' : ''; ?>
+							data-step="<?php echo esc_attr( $timepicker_step ); ?>"
+							data-round="<?php echo esc_attr( $timepicker_round ); ?>"
+							value="<?php echo esc_attr( $ticket_start_time ); ?>"
+							aria-label="
```

Comment:
Needs one-liner treatment (define string at top, echo and escape on one line), specifically no extra whitespace in the attribute quotes.

Reply:
done

Reply:
Not sure why this one isn't showing as changed

Reply:
got it this time. done.